docs/admin_guide/user-administration.rst

```diff
@@ -80,3 +80,51 @@ If a user runs out of projects and wants to create more, they should contact an
    :figclass: align-center
 
    Increase number of projects 
+
+
+Reset users' password
+---------------------
+
+If a user forgets his/her password and security quesion, an administrator can reset it by going to the edit user page as shown above and selecting reset user password button.
```

Comment:
@ErmiasG I suggest to have  `their` or `her` instead of `his/her`

Reply:
@ErmiasG `reset user password` we can have in "`" or italics.

Reply:
@ErmiasG should it read `below` instead of `above`?

Reply:
I was referring to the go to edit page above